source/adapters/level_zero/v2/latency_tracker.hpp

```diff
@@ -0,0 +1,212 @@
+//===--------- ur_latency_tracker.cpp - common ---------------------------===//
```

Comment:
can we move this to `ur_common`? It might be useful for other adapters or even the loader.

Reply:
Sure. Done.